src/components/victory-group/victory-group.js

```diff
@@ -408,9 +408,7 @@ export default class VictoryGroup extends React.Component {
         labels,
         theme: child.props.theme || props.theme,
         labelComponent: props.labelComponent || child.props.labelComponent,
-        domainPadding: child.props.domainPadding
-        || props.domainPadding
-        || {x: (props.offset * childComponents.length) / 2},
+        domainPadding: this.getDomainPadding(props, childComponents, child),
```

Comment:
Looking at what's in `this.getDomainPadding` and what arguments it needs, maybe it makes more sense to have this be something like 

`domainPadding: child.props.domainPadding || props.domainPadding || defaultDomainPadding`

and just pre-calculate `defaultDomainPadding`